packages/opentelemetry-plugin-document-load/src/documentLoad.ts

```diff
@@ -44,7 +44,11 @@ export class DocumentLoad extends BasePlugin<unknown> {
    * callback to be executed when page is loaded
    */
   private _onDocumentLoaded() {
-    this._collectPerformance();
+    // Timeout is needed as load event doesn't have yet the performance metrics for loadEnd.
+    // Support for event "loadend" is very limited and cannot be used
+    window.setTimeout(() => {
+      this._collectPerformance();
+    });
```

Comment:
Would it make sense to have a little delay here, e.g. 1 second or so?

Reply:
if I  understand correctly it should work like this. When load event is being called then the performance is collecting the remaining metrics included "loadEnd", I assume this is synchronous so whenever this is finished the browser will start executing all the timeouts that were set together with the same load event. Then it really doesn't make sense to wait longer as the `loadEnd` metric should be there. If this is not the case then it will  be hard to really say how long you should wait, 100ms, 1s, maybe 2s, but it seems like the moment the setTimeout calls the callback this metric is already there.

Reply:
In the end this is not final, we can always put there 1s at any moment if for any reason it  appeared that some browsers don't have it yet.